lib/v2/caai/utils.js

```diff
@@ -0,0 +1,94 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const url = require('url');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const base = 'http://www.caai.cn';
+
+const urlBase = (caty) => url.resolve(base, `/index.php?s=/home/article/index/id/${caty}.html`); // en, cn, (none, for JP)
```

Comment:
`url.resolve()` [is legacy](https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost). Please use WHATWG URL API instead.